libpod/image/pull.go

```diff
@@ -401,18 +402,19 @@ func (ir *Runtime) pullGoalFromPossiblyUnqualifiedName(inputName string) (*pullG
 	if err != nil {
 		return nil, err
 	}
-	refPairs := make([]pullRefPair, 0, len(searchRegistries))
-	for _, registry := range searchRegistries {
-		ref, err := decomposedImage.referenceWithRegistry(registry)
-		if err != nil {
-			return nil, err
-		}
-		imageName := ref.String()
-		srcRef, err := docker.ParseReference("//" + imageName)
+
+	candidates, err := shortnames.Resolve(nil, inputName, false)
```

Comment:
If nothing else, I love that we finally have the resolution code in c/image so resolving short names will finally be consistent across all our tools.

Reply:
I still hope we'll get a common image library done that Buildah, Podman and CRI-O can share.

Reply:
s/library/package/ :)